src/Microsoft.DotNet.VersionTools/Dependencies/ProjectJsonUpdater.cs

```diff
@@ -133,5 +168,33 @@ private static IEnumerable<JProperty> FindAllDependencyProperties(JObject projec
                 .Select(property => property.Value)
                 .SelectMany(o => o.Children<JProperty>());
         }
+
+        private class DependencyChange
+        {
+            public BuildInfo BuildInfo { get; set; }
+            public string PackageId { get; set; }
+            public NuGetVersion Before { get; set; }
+            public NuGetVersion After { get; set; }
+
+            public override string ToString()
+            {
+                return $"'{PackageId} {Before.ToNormalizedString()}' must be " +
+                    $"'{After.ToNormalizedString()}' ({BuildInfo.Name})";
+            }
+        }
+
+        private class ParsedBuildInfo
+        {
+            public BuildInfo BuildInfo { get; }
+            public IEnumerable<PackageIdentity> Packages { get; }
+
+            public ParsedBuildInfo(BuildInfo buildInfo)
+            {
+                BuildInfo = buildInfo;
+                Packages = buildInfo.LatestPackages
```

Comment:
Ah, yep, I think I didn't realize at this point that I'd want to add that method and I didn't remember to come back here once I did. Fixing.